requirements-dev.txt

```diff
@@ -40,6 +40,7 @@ types-beautifulsoup4
 types-cachetools
 types-croniter
 types-jsonschema
+types-mock
```

Comment:
this is for the deprecated `mock` backport and not for the stdlib `unittest.mock` module

Reply:
OK, I've removed mocking from this.